### PR TITLE
Tidy json

### PR DESCRIPTION
**Pull request by snim2**
Update JSON format and bring related code up to date.

### Test cases (master branch)

  * Bencher7 main table: [master_bencher7_table.pdf](https://github.com/softdevteam/warmup_experiment/files/961995/master_bencher7_table.pdf)
  * Octane bencher7 table: [master_octane_bencher7_table.pdf](https://github.com/softdevteam/warmup_experiment/files/961997/master_octane_bencher7_table.pdf)
 * Octane bencher7 HTML table: (crashes)

### Test cases (feature branch)

  * Example JSON: https://gist.github.com/snim2/d9e365e81a54a318f2f3c0840dd74af7
  * Bencher7 main table: [feature_bencher7_table.pdf](https://github.com/softdevteam/warmup_experiment/files/962007/feature_bencher7_table.pdf)
  * Octane bencher7 table: [feature_octane_table.pdf](https://github.com/softdevteam/warmup_experiment/files/962009/feature_octane_table.pdf)
 * Octane bencher7 HTML table: feature_bencher7_octane

### CODE REVIEW

File: warmup/summary_statistics.py [hunk 1]
```diff
@@ -410,9 +417,8 @@ def write_html_table(summary_data, html_filename):
             else:
                 mean_steady_iter = ''
             if bmark['steady_state_time'] is not None:
-                mean_steady = '%.5f (%.5f, %.5f)' % (bmark['steady_state_time'],
-                                                     bmark['steady_state_time_iqr'][0],
-                                                     bmark['steady_state_time_iqr'][1])
+                mean_steady = '%.5f&plusmn;%.6f' % (bmark['steady_state_time'],
```
Comment (vext01) [inline warmup/summary_statistics.py:420]: I'm 99% sure it was, but was it intentional to change from IQRs to CIs?

**Comment by vext01** [comment]
I don't see anything dodgy here.

File: warmup/summary_statistics.py (hunk 1 above)
Comment (snim2) [reply warmup/summary_statistics.py:420]: yes, but only for the **steady state perf** columns of the tables. This was the cause of the HTML table script crashing in the master branch.

**Comment by ltratt** [comment]
Looks good to me!

**Comment by ltratt** [comment]
Do you want to squash and then I can merge?

**Comment by snim2** [comment]
No need to squash, I think

**Review by ltratt — APPROVED** [review]

**Comment by ltratt** [comment]
@vext01 ok for me to merge?

**Comment by vext01** [comment]
sure.

**Pull request merged**